src/components/setting/setting.test.js

```diff
@@ -46,12 +45,17 @@ describe('Setting', () => {
 
   const t = key => key;
   let wrapper;
+
   const settingsUpdated = sinon.spy();
+  const accountUpdated = sinon.spy();
```

Comment:
You can define `let accountUpdated` here, but `sinon.spy();` should be inside of `beforeEach`. The current way, the same spy is used in all `it`s, so the tests are not isolated and can affect each other.